File: Just_codes/Python/largestof_three_num.py

def max(num1,num2,num3):
    if(num1>num2) and (num1>num3):
        large = num1
    elif(num2>num1) and (num2>num3):
        large = num2
    else:
        large = num3
    print('The largest of three number is ',large)
    
def min(num1,num2,num3):
    if(num1<num2) and (num1<num3):
        small = num1
    elif(num2<num1) and (num2<num3):
        small = num2
    else:
        small = num3
    print('The largest of three number is ',small)

n1=int(input('Enter a first number: '))
n2=int(input('Enter a second number: '))
n3=int(input('Enter a third number: '))

max(n1,n2,n3)
min(n1,n2,n3)

File: Just_codes/C++/LargestOfThree_Num.cpp
#include <iostream>
using namespace std;

int large_num(int x, int y, int z);

int main(){
    
    int x,y,z;
    cout << "Enter a 1st Number: ";
    cin >> x;
    cout << "Enter a 2nd Number: ";
    cin >> y;
    cout << "Enter a 3rd Number: ";
    cin >> z;
    
    int largest_number = large_num(x,y,z);
    cout << "Largest number: " << largest_number << endl;

}

int large_num(int x, int y, int z){
    
    if(x > y && x > z){
        return x;
    }
    else if(y > x && y > z){
        return y;
    }
    else{
        return z;
    }
}